### ISSUE TITLE
[app name] compatibility report

### ISSUE THREAD

**Issue by bege10** [AdAway/AdAway#2873]
**Describe the third party app**
 - Name: [DB Navigator, browsers: Privacy, Fennec]
 - Version: [DB Navigator 21.06.p04.00]

**Describe the compatibility issue**
When the root web server is active some functions of DB Navigator throw an error that a certificate for a secure connection is missing.
Browsers throw a security warning when displaying the web server test.

**Workaround**
- allow all related tracking URLs or
- better: disable web server.

Can this issue be solved in a way that redirecting secure connections to the local web server does not throw these errors?

`workaround-available`

**Comment by dreamflasher** [comment]
How do these workarounds work? What are specifically all related tracking URLs for the DB navigator? 
How do I disable the web server?

**Comment by bege10** [comment]
> 
> 
> How do these workarounds work? What are specifically all related tracking URLs for the DB navigator?

To find out for your device
1. Stop AdAway (red button at bottom of screen)
2. Turn on DNS log
3. Start DB Navigator
4. See URLs in DNS log

> How do I disable the web server?

1. Go to AdAway settings
2. tap on (in case of rooted device) Root based adblocker
3. deactivate web server (at bottom of the screen)

**Comment by Larx** [comment]
Has anybody found out which URLs should be enabled to get DBNavigator to work? This is really annoying :-( ...

**Comment by bege10** [comment]
----------------------------------------


> problem is still existing. it took me ages to find adaway as the culprit -_-
> at least "disable web server" solves the problem.
> is there any disadvantage to this?
> 

As far as I understand the server is only for the case that an app crashes when it cannot reach a blocked IP.
I didn't have problems yet without the server.

**Comment by mayonezo** [comment]
This is a duplicate of issue [#2427](https://github.com/AdAway/AdAway/issues/2427)

**Comment by dreamflasher** [comment]
I agree, this one here can be closed. Also deactivating the local web server solves it.
